docs/quick-start-docker-cluster.md

```diff
@@ -0,0 +1,165 @@
+# Quick start for docker-cluster
+
+The docker-cluster image is similar to docker-standard, but with the following changes and features:
+
+* Cluster of 4 metrictanks (with 2x replication)
+* Separate Graphite monitoring server
+* The metrictank binary is **not** baked in
```

Comment:
it is (same image as elsewhere), but to be more precise, we mount the binary from the `build` directory in to it, so must run `make bin` before starting the stack.

Reply:
This was an issue for me when first trying to run docker-cluster. How would you feel about removing line 7 and adding a note about why the metrictank binary is mounted further down the document(to speed up development / testing)?

Reply:
i'm ok with having the note here, just make it more precise (and feel free to mention why)